packages/core-mobile/app/new/features/accountSettings/components/AccountItem.tsx

```diff
@@ -50,6 +51,13 @@ export const AccountItem = memo(
     } = useTheme()
     const { formatCurrency } = useFormatCurrency()
 
+    const { handleParentPress, createChildPressHandler } =
+      usePreventParentPress(() => onSelectAccount(account))
+
+    const handleChildPress = createChildPressHandler(() =>
+      gotoAccountDetails(account.id)
+    )
+
```

Comment:
I think it's better to also have a createParentPressHandler instead of passing the parent press logic as a hook param